### PR TITLE
Minor changes to tailwind docs

### PR DESCRIPTION
**Pull request by themmes**
- append to gitignore with newline
- remove inconvenient syntax highlighting for makefiles
- add alternative to run `make tailwind-dev` and `RunDevServer` together

### CODE REVIEW

File: Guide/tailwindcss.markdown [hunk 1]
```diff
@@ -114,7 +114,7 @@ form label {
 
 We need to add a new build command for starting a tailwind build process to our `Makefile`. For that append this to the `Makefile` in your project:
 
-```makefile
```
Comment (mpscholten) [inline Guide/tailwindcss.markdown:117]: I guess we a better solution would be to fix the css instead :) Likely we just need to add a `, .language-makefile` to this css rule here: https://github.com/digitallyinduced/ihp/blob/master/Guide/layout.html#L194

**Comment by mpscholten** [comment]
Any updates?

**Comment by themmes** [comment]
That should do it!

**Review by mpscholten — APPROVED** [review]

**Comment by mpscholten** [comment]
Perfect, thanks :)

**Pull request merged**